docs/developer/installation.rst

```diff
@@ -16,13 +16,21 @@ Install sqlite:
 
     sudo apt-get install sqlite3 libsqlite3-dev openssl libssl-dev
 
-Install your forked repo:
+Create and activate virtual environment in your forked repo:
 
 .. code-block:: shell
 
     git clone git://github.com/<your_fork>/openwisp-ipam
     cd openwisp-ipam/
+    virtualenv venv
+    source venv/bin/activate
+
+Install your forked repo:
+
+.. code-block:: shell
+
     pip install -e .
+    
```

Comment:
Can you make these steps consistent with what we have in [openwisp-controller](https://github.com/openwisp/openwisp-controller/blob/455144437b947fa343e39e0cea33ca997782f369/docs/developer/installation.rst?plain=1#L46-L52)?